### PR TITLE
#162967270 Log In page for Questioner (UI)

### PR DESCRIPTION
**Pull request by Ogutu-Brian**
### What does this PR do?
Creates a log in page for Questioner
### Description of Task to be completed?
Have the following tasks completed
1. Have a responsive design
2. Have a simple User Interface
3. Have matching color theme
### How should this be manually tested?
After cloning the repo, cd into UI and click `login.html`
### Screenshots
screenshot from 2019-01-04 23-12-05
screenshot from 2019-01-04 23-12-28

### What are the relevant pivotal tracker stories?
[#162967270](https://www.pivotaltracker.com/story/show/162967270)

### CODE REVIEW

File: UI/login.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+            <input type="password"><br><br>
+            <input type="button" class="button" value="Signup">
+        </form>
+    </div>
+    <footer>
+        &copy;Questioner 2019
+    </footer>
+</body>
+
+</html>
```
Comment (Ogollah) [inline UI/login.html:38]: new line

**Comment by Ogollah** [comment]
simple and nice.

File: UI/login.html (hunk 1 above)
Comment (Ogutu-Brian) [reply UI/login.html:38]: Thanks, what is the importance of the new line?

**Comment by Ogutu-Brian** [comment]
> simple and nice.

Thanks

**Pull request merged**